docs/spec-files/requirements.md

```diff
@@ -2,7 +2,7 @@
 
 The [DIDComm Messaging](.) design attempts to be:
 
-1. **Secure** (tamper-proof; uses best-of-breed crypto; allows parties to talk both on and off the record.)
+1. **Secure** (specifically, MUST preserve the confidentiality of messages against eavesdropping, MUST preserve the integrity of messages against tampering, and MUST allow the authenticity of messages and message senders to be proved; MUST use best-of-breed crypto; MUST allows parties to emit both repudiable and non-repudiable messages; perfect forward secrecy is not formally required due to the lack of a session construct, but similar outcomes must be achievable, as described in the [Implementer's Guide](guide.md#pfs))
```

Comment:
Looks better. On thing 

Maybe i am wrong, but eavesdropping protection is usually property of privacy.

Confidentially is usually ethical duty of information recipient to don’t re-share information he got from sender. 

For example `doctor` should keep `patient` information confidential and keep place of medical examination private.

Some protocols can support confidentially by nature. For example, CL-signatures.

In DID Comm i think we can only achieve absence of additional threats to confidentiality like unexpected non-repudiation.